src/slack/utilities/getDashboardContext.ts

```diff
@@ -7,10 +7,10 @@ interface DashboardContext {
 
 export async function getDashboardContext(userId: string): Promise<DashboardContext> {
   const teamRegistrationToggle = (await Config.findOne({ key: 'teamRegistrationActive' })) || { key: 'teamRegistrationActive', value: 'false' };
-  const subscriber = await Subscriber.findOne({ slackId: userId, isActive: true });
+  const isSubscribed = (await Subscriber.count({ slackId: userId, isActive: true })) > 0;
```

Comment:
For this, can we have just the await? I feel doing async and sync on the same line can get confusing.